File: client-app/src/app/models/LevelTest.ts
import { StudentTestClass } from "./StudentTest";

export interface ILevelTest
{
    id:string;
    name:string;
    groupTestId:string;
    groupTest:string;
    test:string;
    studentTestId:string;
    studentTest:string;
    timeToTest: string;
    students:StudentTestClass[];
}

export class LevelTestClass implements ILevelTest {
  id:string="";
  name:string="";
  groupTestId:string="";
  groupTest:string="";
  test:string="";
  studentTestId:string="";
  studentTest:string="";
  timeToTest: string="";
  students:StudentTestClass[]=[];
    constructor(init?: LevelTestClass) {
      Object.assign(this, init);
    }
  }

File: Domain/LevelTest.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class LevelTest
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public Guid groupTestId { get; set;}
        public GroupTest groupTest { get; set; }
        public string TimeToTest { get; set; }


        //идентификатор теста, в базе нету такого поля
        //public string studentTestId { get; set; }
        // public List<Test> Test {get;set;}
        // public List<StudentTest> StudentTest {get;set;}
        
    }
}